### DISCUSSION TITLE
aplus-ui/components/ap-download/

### DISCUSSION THREAD

**Discussion by giscus** [ollieLiu1988/aplus-ui-comments#8] [General]
# aplus-ui/components/ap-download/

Aplus basic doc site.

https://aplus.aplnk.com/aplus-ui/components/ap-download/

**Comment by gofollowmymaster** [comment]
升级到新版本（0.1.40）后当设置disabled="true"  样式出现了意外的边框及背景色，查看dom节点显示，是button按钮样式

**Comment by gofollowmymaster** [comment]
###   ApDownload样式优化 【优化】    
版本：0.1.40
#### 背景&原因
使用ApDownload 组件，设置disabled 为true后样式出现了问题
#### 需求描述
1. 设置disabled 为true后 默认样式能保持文本模式

**Reply by ollieLiu1988** [reply]
![1730685816439](https://github.com/user-attachments/assets/c0427ad2-7f0d-4a2d-bd2a-2af7e3c25266)
组件库里是正常的，请检查项目中的全局样式覆盖

**Reply by gofollowmymaster** [reply]
确认是全局覆盖样式原因导致的
